Cargo.toml

```diff
@@ -10,10 +10,27 @@ include = ["**/*.rs", "Cargo.toml", "README.md", ".gitignore"]
 
 [dependencies]
 blake2 = "0.10.6"
-rand_core = "0.6.4"
 
 [dev-dependencies]
+rand = "0.8.5"
+rand_core = "0.6.4"
 rand_chacha = "0.3.1"
+criterion = { version = "0.5.1", features = ["html_reports"]}
+
+[[bench]]
+name = "centralized_time"
+harness = false
+path = "benches/centralized_telescope.rs"
+
+[[bench]]
+name = "centralized_step"
+harness = false
+path = "benches/centralized_telescope.rs"
+
+[[bench]]
+name = "centralized_repetitions"
```

Comment:
I agree with Tolik. We shouldn't have this warning. However, as the project grows there will be many files in the `benches` folder. So, we could have the following:
- `/benches/criterion_helpers.rs`: Common code
- `/benches/step.rs`: Step benchmark, handling step benches for each setup.
- `/benches/time.rs`: Time benchmark, handling time benches for each setup.
The file names could be more descriptive of course.

Reply:
Perhaps we could have a subfolder per protocol?

Reply:
Definitely! Suggest `benches/centralized_telescope/` under which we would have different files for different benchmarks plus auxiliary files.